app/src/components/Footer.tsx

```diff
@@ -1,4 +1,4 @@
-import { useTranslation } from "next-i18next";
+import { useTranslations } from "next-intl";
```

Comment:
nit: Could we re-export `useTranslations` within the `src/i18n` module and import `useTranslations` from `src/i18n` rather than directly importing the library?

Might feel nitpicky but generally except in some cases for framework dependencies (i.e. Next.js) or very light utility dependencies like lodash, I prefer to wrap all dependencies in our own module and re-export only the functionality we want to use.

The rationale is that we generally want to control our dependencies, and when we expand our dependencies we want that decision to be an explicit one that is reviewed and critiqued rather than one that a single engineer can accidentally slip by. For adding entire libraries, this is caught by changes to package.json — if a reviewer sees a new package in package.json that wasn't previously discussed, they'll easily catch that and it's an opportunity to discuss the addition of the dependency. Within a package, there is typically a lot of functionality since libraries are designed to be maximally flexible and support a lot of use cases across a variety of projects. However, on our specific project, you typically only want a specific subset of that functionality, and you want any changes to the subset of functionality you depend on to be an explicit decision. If the pattern is for engineers to import libraries directly, then it's very easy to start using functionality in the library that the original engineer that added the library didn't intend to use. Therefore, I think it's best to explicitly wrap the functionality you want in a wrapper module, and have engineers only import the wrapper module. This way, if an engineer wants to use additional functionality from the library, they'll have to explicitly add it to the wrapper module, and that will be more easily caught in review to ensure that it's an explicit decision to do that (and incur the associated cost of expanding the surface area of dependencies).

For example it could be like

```js
// i18n.js
export { useTranslations } from "next-intl";
```

then here we'd do
`import {useTranslations} from "../src/i18n"` instead of `from "next-intl"`